services/youtube/youtube-channel-base.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService, NotFound } = require('..')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+
+const documentation = `
+<p>By using the YouTube badges provided by Shields.io, you are agreeing to be bound by the YouTube Terms of Service. These can be found here:
+<code>https://www.youtube.com/t/terms</code></p>`
+
+const schema = Joi.object({
+  pageInfo: Joi.object({
+    totalResults: nonNegativeInteger,
+    resultsPerPage: nonNegativeInteger,
+  }).required(),
+  items: Joi.array().items(
+    Joi.object({
+      statistics: Joi.object({
+        viewCount: nonNegativeInteger,
+        subscriberCount: nonNegativeInteger,
+        hiddenSubscriberCount: Joi.boolean().required(),
```

Comment:
No need to validate on `hiddenSubscriberCount` and `videoCount` if we're not actually using them.